src/components/playground/PlaygroundComponent/PlaygroundComponent.jsx

```diff
@@ -31,6 +33,28 @@ export const PlaygroundComponent = () => {
   const [selected, setSelected] = useState("Select API Call - Version 3");
   const location = useLocation();
   const history = createBrowserHistory();
+  
+  // If the user switches to a different tab, it will trigger the visibility state.
+  const documentVisibility = () => {
+    // If the visibility state is hidden, we will close the API.
+    if (document.visibilityState === "hidden") {
+      current_api.connection.close();
+      ticksSubject.complete();
+      document.removeEventListener("visibilitychange", documentVisibility, false);
```

Comment:
capture is false by default, so in our case we can remove third parameter in removeEventListener

Reply:
Good point. Will remove it.